middleware/logger_test.go

```diff
@@ -91,17 +91,17 @@ func TestLoggerTemplate(t *testing.T) {
 	e.Use(LoggerWithConfig(LoggerConfig{
 		Format: `{"time":"${time_rfc3339_nano}","id":"${id}","remote_ip":"${remote_ip}","host":"${host}","user_agent":"${user_agent}",` +
 			`"method":"${method}","uri":"${uri}","status":${status}, "latency":${latency},` +
-			`"latency_human":"${latency_human}","bytes_in":${bytes_in}, "path":"${path}", "referer":"${referer}",` +
+			`"latency_human":"${latency_human}","bytes_in":${bytes_in}, "path":"${path}", "route":"${route}" "referer":"${referer}",` +
```

Comment:
to be pedantic :)  `,` comma is missing between `route` and `referer`

Reply:
Oh thanks for the good catch, fixed🙏
https://github.com/labstack/echo/pull/2162/commits/10a35c5b3c3d4002250d65a86eb349b0c363e949